### ISSUE TITLE
Backups

### ISSUE THREAD

**Issue by IndianaTones** [KatyaAplonova/ReportedDiscourseProject#19]
@rvoll - your backups aren't working :( I don't have any backed up data for you on the hard-drives here.

**Comment by rvoll** [comment]
Hi Abbie! What do you mean, they aren't working? Maybe it's because I cloned my hard drive. Shoudl we discuss this on skype? I think that will be easier. Maybe tomorrow after the PhD search meeting or after the ELAN meeting? Or just write me when you get around to it and we chat about it. Okay?

**Comment by IndianaTones** [comment]
It's because they need to be formatted for both PC and Mac. You can just do it next time you come in to the office.

**Comment by rvoll** [comment]
Okay. So it's there, just not for PC. I'll try to fix it next week!

**Comment by IndianaTones** [comment]
Yes, I was able to see them on my Mac but not transparently. Would be better to just have a simple data backup rather than the disk clones.
